vagrant-kubernetes-istio/README.md

```diff
@@ -69,7 +69,7 @@ You can try debugging Istio with debugger tool. vm_setup.sh already installs Del
 For example, if you want to debug a test, run that test in your host/vm.
 ```bash
 # In the VM/Host
-make e2e_simple E2E_ARGS="--use_local_cluster --skip_cleanup"
+make e2e_simple E2E_ARGS="--use_local_cluster --skip_cleanup" HUB=10.10.0.2:5000 TAG=latest
```

Comment:
If you don't pass HUB and TAG here, it may not work. Although we set HUB and TAG in setup_test.sh, $ sh setup_test.sh does not set global environment HUB and TAG for us. Maybe we can run $. setup_test.sh to make them global environment. I had issue when I ran make e2e_simple E2E_ARGS="--use_local_cluster"